sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -124,10 +124,39 @@ def __init__(
         self.require_encryption = kwargs.get("require_encryption", False)
         self.key_encryption_key = kwargs.get("key_encryption_key")
         self.key_resolver_function = kwargs.get("key_resolver_function")
-        self._config, self._pipeline = self._create_pipeline(
-            self.credential, storage_sdk=service, **kwargs
+
+        self._configure_credential(self.credential)
+
+        self._config = kwargs.get("_configuration") or AzureTableConfiguration(
+            url=self.url,
+            headers_policy=StorageHeadersPolicy(**kwargs),
+            user_agent_policy=UserAgentPolicy(sdk_moniker=SDK_MONIKER, **kwargs),
+            retry_policy=kwargs.get("retry_policy") or TablesRetryPolicy(**kwargs),
+            logging_policy=StorageLoggingPolicy(**kwargs),
+            proxy_policy=ProxyPolicy(**kwargs)
         )
 
+        kwargs.setdefault("connection_timeout", CONNECTION_TIMEOUT)
+        kwargs.setdefault("read_timeout", READ_TIMEOUT)
+        self._config.transport = kwargs.get("transport") or RequestsTransport(**kwargs)
```

Comment:
transport is a property on the pipeline, not on the config. [Here's][https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/azure/core/_pipeline_client.py#L101] the code for the pipeline client, and it appears to do this code for you, so you shouldn't need this line, just make sure you pass the kwargs to the generated client and this shoudl be handled for you